lib/rucio/tests/test_download.py

```diff
@@ -48,40 +54,271 @@ def setUp(self):
         logger.addHandler(logging.StreamHandler())
         logger.setLevel(logging.DEBUG)
         self.client = Client()
+        self.did_client = DIDClient()
```

Comment:
Only a minor detail the `Client()` includes all clients, except the `UploadClient` and `DownloadClient` - so you do not explicitly need to import it.